mods/d2k/rules/defaults.yaml

```diff
@@ -131,13 +123,16 @@
 	TargetableUnit:
 		TargetTypes: Ground
 		RequiresForceFire: yes
-	Capturable:
-		Type: husk
-		AllowAllies: yes
-		CaptureThreshold: 1.0
-	TransformOnCapture:
-		ForceHealthPercentage: 25
+#	Capturable:
+#		Type: husk
+#		AllowAllies: yes
+#		CaptureThreshold: 1.0
+#	TransformOnCapture:
+#		ForceHealthPercentage: 25
```

Comment:
Our policy is to not leave commented out lines in, but rather delete them, especially in cases like this (unless someone explicitly said otherwise on IRC).